components/script/dom/webidls/MutationObserver.webidl

```diff
@@ -9,7 +9,8 @@
 // https://dom.spec.whatwg.org/#mutationobserver
 [Pref="dom.mutation_observer.enabled", Constructor(MutationCallback callback)]
 interface MutationObserver {
-    //void observe(Node target, optional MutationObserverInit options);
+    [Throws]
```

Comment:
Why the `Throws` marker? It's not in the whatwg idl: https://dom.spec.whatwg.org/#mutationobserver